packages/camera/camera_android/android/gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.jvmargs=-Xmx16Gheheheheheh
```

Comment:
What's the `heheheheheh`?

Reply:
This was a test to see if I could change the heap size this way. It didn't impact anything and I forgot to remove it -- oops!